src/lib/Scenes/Show/Components/ShowHeader/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+
+import { Box, Separator, Serif, space, Spacer } from "@artsy/palette"
+import { ShowHeader_show } from "__generated__/ShowHeader_show.graphql"
+import { InvertedButton } from "lib/Components/Buttons"
+import { CaretButton } from "lib/Components/Buttons/CaretButton"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { Carousel } from "./Components/Carousel"
+import { Chip } from "./Components/Chip"
+
+interface Props {
+  show: ShowHeader_show
+  onSaveShowPressed: () => Promise<void>
+  onMoreInformationPressed: () => void
+}
+
+export class ShowHeader extends React.Component<Props> {
+  render() {
+    const {
+      show: { images, name, exhibition_period, status_update, description },
+      onSaveShowPressed,
+      onMoreInformationPressed,
+    } = this.props
+    return (
+      <>
+        <Carousel
+          sources={(images || []).map(({ url: imageURL, aspect_ratio: aspectRatio }) => ({ imageURL, aspectRatio }))}
+        />
+        <Serif size="5t" weight="semibold" textAlign="center">
+          {name}
+        </Serif>
+        <Serif size="3" textAlign="center">
+          {exhibition_period}
+        </Serif>
+        <Box px={space(2)}>
```

Comment:
Is the full component tree wrapped in a `<Theme>` component from `@artsy/palette`? If so these values can be pulled in directly via styled-components' theme, e.g., 

```js
<Box px={2}>
```